tests/cypress/integration/regTest_for_export_task/innerExportTask.js

```diff
@@ -0,0 +1,138 @@
+
+    if (imageClick) {
+        cy.get('.cvat-modal-export-save-images').click();
+    }
+
+    cy.get('.cvat-modal-export-task').contains('button', 'OK').click();
+
+    cy.get('.cvat-notification-notice-export-task-start').should('be.visible');
+    cy.closeNotification('.cvat-notification-notice-export-task-start');
+    cy.getDownloadFileName().then((file) => {
+        cy.verifyDownload(file);
+        cy.get('.ant-notification-notice-info').first().should('be.visible');
+        cy.closeNotification('.ant-notification-notice-info');
+    });
+});
+
+context('Export task inner.', { browser: '!firefox' }, () => {
+    const caseId = '999';
+    const labelName = 'block';
+    const taskName = `Case test inner ${caseId}`;
+    const attrName = 'color';
+    const textDefaultValue = 'red';
+    const imagesCount = 1;
+    const imageFileName = `image_${taskName.replace(/\s+/g, '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    const newLabelName = 'person';
+    const exportOrders = [0, 1, 2];
+
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName,
+        firstX: 250,
+        firstY: 350,
+        secondX: 350,
+        secondY: 450,
+    };
+
+    before(() => {
+        cy.visit('auth/login');
+        cy.login();
+        cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+        cy.createZipArchive(directoryToArchive, archivePath);
+        cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName);
+        cy.openTask(taskName);
+        cy.addNewLabel(newLabelName);
+        cy.openJob();
+        cy.createRectangle(createRectangleShape2Points).then(() => {
+            Cypress.config('scrollBehavior', false);
+        });
+        cy.get('#cvat_canvas_shape_1')
+            .trigger('mousemove')
+            .trigger('mouseover')
+            .should('have.class', 'cvat_canvas_shape_activated');
+        cy.get('.svg_select_points_rot')
+            .should('be.visible')
+            .and('have.length', 1)
+            .trigger('mousemove')
+            .trigger('mouseover');
+        cy.get('.svg_select_points_rot').trigger('mousedown', { button: 0 });
+        cy.get('.cvat-canvas-container').trigger('mousemove', 340, 150);
+        cy.get('.cvat-canvas-container').trigger('mouseup');
+        cy.get('#cvat_canvas_shape_1').should('have.attr', 'transform');
+        cy.saveJob();
+        cy.goToTaskList();
+    });
+
+    after(() => {
+        cy.goToTaskList();
+        cy.deleteTask(taskName);
+    });
+
+    describe(`Testing "${taskName}"`, () => {
```

Comment:
Better to be more specific here. It is not clear what we are testing. Using string template is not the best option here because it makes code more difficult to read and understand.